src/features/CellEditing.js

```diff
@@ -3,12 +3,39 @@
 var Feature = require('./Feature');
 var CellEditor = require('../cellEditors/CellEditor');
 
+var KEYS = {
+    RETURN: 'RETURN',
+    RETURNSHIFT: 'RETURNSHIFT',
+    DELETE: 'DELETE',
+    BACKSPACE: 'BACKSPACE',
+    SPACE: 'SPACE',
+    F2: 'F2'
+};
+
 /**
  * @constructor
  * @extends Feature
  */
 var CellEditing = Feature.extend('CellEditing', {
 
+    isVisibleChar: function(char, event) {
+        return !!(
```

Comment:
`!!` pointless here

Reply:
Yup, it'll be removed when the code is refactored back to handleKeyDown.